Source/AwsCommonRuntimeKit/Utilities.swift

```diff
@@ -46,6 +46,13 @@ extension String {
     }
 }
 
+extension aws_date_time {
+    func toDate() -> Date {
+        let timeInterval = withUnsafePointer(to: self) { aws_date_time_as_epoch_secs($0) }
```

Comment:
Nit: You can pass simply pass in the function when the function matches the closure signature

```suggestion
        let timeInterval = withUnsafePointer(to: self, aws_date_time_as_epoch_secs)
```